Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM ubuntu:20.04
+FROM ubuntu:23.04
 
-ARG MUSE_VERSION=2.0.4
-ARG MUSE_SHA512=8342e648d389b7feef04f870c48bcddbbbdfc85e0c5faf261e6bf68e2f48a02be3e3251691004cf7bb0164de25dc1ef8323dbe5eabc040ba54740c034337a997
```

Comment:
**comment (blocking)**: We should really add this back in

Reply:
Done and tested:
`/hot/code/sfitzgibbon/GitHub/uclahs-cds/docker-MuSE/slurm-231091.out`